src/aish/shell/ui/prompt_io.py

```
@@ -366,6 +367,39 @@ def _prepare_interaction_prompt(shell: Any) -> None:
+        yield
+        return
+
+    stdin_yield_ack_event = getattr(shell, "_stdin_yield_ack_event", None)
+    wake_stdin_monitor = getattr(shell, "_wake_stdin_monitor", None)
+
+    stdin_yield_event.set()
+    if callable(wake_stdin_monitor):
+        try:
+            wake_stdin_monitor()
+        except Exception:
+            pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)